code/style.css

```diff
@@ -1,4 +1,207 @@
+/* GLOBAL STYLES */
```

Comment:
Good job by leaving comments like these! It is easier to read and find the code that you need :)